src/quickstart/provider/azure-acs/boot.md

```diff
@@ -167,27 +168,16 @@ The Kubernetes cluster will take a few minutes to complete provisioning and conf
 
 ## Connect to the ACS Kubernetes Cluster
 
-Retrieve the fully qualified domain name (FQDN) for the Kubernetes master.
+`kubectl` is the Kubernetes command line client.  If you don't already have it installed, you can install it with:
 
-```
-$ export K8S_FQDN=`az acs list -g $RG_NAME --query [0].masterProfile.fqdn --output tsv`
-$ echo $K8S_FQDN
-```
-
-Download the Kubeconfig from the master to the local machine, make sure to use the same SSH credentials used to create the cluster:
-
-```
-$ scp -i ~/.ssh/id_rsa k8sadmin@$K8S_FQDN:.kube/config ~/.kube/k8sanddeis.config
-The authenticity of host 'mydnsprefix.myregion.cloudapp.azure.com (40.78.71.181)' can't be established.
-ECDSA key fingerprint is a0:09:ff:59:83:47:70:38:d4:0d:68:b2:cf:0f:2a:cf.
-Are you sure you want to continue connecting (yes/no)? yes
-Warning: Permanently added 'mydnsprefix.myregion.cloudapp.azure.com,40.78.71.181' (ECDSA) to the list of known hosts.
+```console
+az acs kubernetes install-cli
 ```
 
-Point `kubectl` at the kubernetes configuration file by setting the `KUBECONFIG` environment value:
+Download the master kubernetes cluster configuration to the ~/.kube/config file by running the following command:
 
-```
-export KUBECONFIG=~/.kube/k8sanddeis.config
+```console
+az acs kubernetes get-credentials --resource-group=$RG_NAME --name=$SERVICE_NAME
```

Comment:
@dtzar Ah, this is great! What version of the `az` CLI do you need for this command to work?